pkg/pillar/zedUpload/datastore_req.go

```diff
@@ -66,6 +68,15 @@ type DronaRequest struct {
 
 	// Download counter
 	retry int
+
+	// used by Multipart upload
+	Adata     []byte
+	PartID    string
+	SasURI    string
+	Container string
```

Comment:
Container/Bucket is already available in the datastore specific structure. Please remove this.
Ref: https://github.com/zededa/zedcloud/blob/9987cacba846c95d71c6de40fecd08c9b33910cc/vendor/github.com/lf-edge/eve/pkg/pillar/zedUpload/datastore_azure.go#L170